sophia/src/graph/_traits.rs

```diff
@@ -358,6 +358,9 @@ pub trait Graph {
         Ok(res)
     }
 
+    /// Number of triples stored in the graph.
+    fn len(&self) -> usize;
+
```

Comment:
The omission of a `len` method was deliberate. I didn't want to impose to all implementations that they should be able to give an exact number of triples. We can still discuss that choice, though.

The "recommended" way of estimating the number of triples would be to call `g.triples().size_hint()`. Granted, this should be documented... :)

Reply:
Alright, I'll fix that